charts/catalog/templates/apiserver-deployment.yaml

```diff
@@ -1,3 +1,63 @@
+kind: ServiceAccount
```

Comment:
It probably does make more sense to shove the rbac stuff into the deployments that need it set up. 

Is there any guidance on the difference or preference for multi-yaml split by `---` or using a    `List` object?